### ISSUE TITLE
Discord RPC/last.fm support

### ISSUE THREAD

**Issue by itsWindows11** [huynhsontung/Screenbox#115]
It would be good to have an option to show the media you're currently listening to on Discord, and/or have last.fm integration.

**Comment by huynhsontung** [comment]
Good idea. I haven't thought of Discord but I hope we can get last.fm integration going at some point probably after online media metadata look-up feature is implemented.
